utils/src/main/java/org/apache/cloudstack/utils/security/SSLUtils.java

```diff
@@ -39,7 +39,26 @@
             }
             set.add(s);
         }
-        return (String[]) set.toArray(new String[set.size()]);
+        return set.toArray(new String[set.size()]);
+    }
+
+    /**
+     * It returns recommended protocols that are considered secure.
+     */
+    public static String[] getRecommendedProtocols() {
+        String[] supportedProtocols = { "TLSv1", "TLSv1.1", "TLSv1.2" };
```

Comment:
This is a little cosmetic, but, how about returning directly the array? Instead of assigning it to a variable first?
Also, what about test cases? I mean, if someone starts using this method because it expected the behavior of returning { "TLSv1", "TLSv1.1", "TLSv1.2" }, but later some other person changes it to return only { "TLSv1.1", "TLSv1.2" }, this might affect people using this method